java-client/src/main/java/org/opensearch/client/opensearch/_types/ErrorStringResponse.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```

Comment:
New code or copied from ES?

If new, remove this part. 
If from ES, confirm that it's APLv2 please.

Reply:
Removed.

Reply:
Remove the `Modifications Copyright OpenSearch...` part too. We only need the SPDX header.

Reply:
Done